staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -154,6 +154,8 @@ type DiscoveryClient struct {
 	restClient restclient.Interface
 
 	LegacyPrefix string
+	// Forces the client to accept only "unaggregated" (legacy) discovery.
```

Comment:
```suggestion
	// Forces the client to request only "unaggregated" (legacy) discovery.
```

Reply:
eh, I guess the client requests via the Accept header so it's technically correct either way